lib/movie_wager_api/repo.ex

```diff
@@ -1,3 +1,43 @@
 defmodule MovieWagerApi.Repo do
   use Ecto.Repo, otp_app: :movie_wager_api
 end
+
+
+defmodule MovieWagerApi.Repo.Tasks do
+
+### Release Tasks:
+
+  def migrate do
+    load_app()
+    start_repo(MovieWagerApi.Repo)
+    migrations_path = Application.app_dir(:movie_wager_api, "priv/repo/migrations")
+    # Comment
+    info "Executing migrations for #{inspect MovieWagerApi.Repo} in #{migrations_path}:"
```

Comment:
We only seem to use the `info` method here. Can we just change this to `IO.puts` and get rid of `info`?